Documentation/network/concepts/ipam/eni.rst

```diff
@@ -509,17 +509,11 @@ If ``--instance-tags-filter`` is used:
 EC2 instance types ENI limits
 *****************************
 
-Currently the EC2 Instance ENI limits (adapters per instance + IPv4/IPv6 IPs per adapter) are
-hardcoded in the Cilium codebase for easy out-of-the box deployment and usage.
+The EC2 Instance ENI limits is only fetched from the EC2 API dynamically from 1.18 onwards.
 
-The limits can be modified via the ``--aws-instance-limit-mapping`` CLI flag on
-the cilium-operator. This allows the user to supply a custom limit.
+This requires the EC2 having ``DescribeInstanceTypes`` IAM permission, which is included in the EKS built-in policy ``AmazonEKSWorkerNodePolicy``.
```

Comment:
minor suggestion: I like linking to cloud provider docs when talking about their managed stuff, in this cases we can link to https://docs.aws.amazon.com/aws-managed-policy/latest/reference/AmazonEKSWorkerNodePolicy.html